README.rst

```diff
@@ -1,7 +1,7 @@
 ==========
 EOL Notice
```

Comment:
Do we want to say End of Life Notice instead of EOL Notice to keep it consistent with others?

Reply:
+1

Reply:
Updated, thank you for catching me!